#An array of poses that represents a global path
Header header
float32 resolution
geometry_msgs/PointStamped[] points
